home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/ContactService.java

```diff
@@ -1,13 +0,0 @@
-package com.softserveinc.ita.homeproject.homeservice.service;
```

Comment:
I think that service packages should be organized like repo packages in home-data, i.e.:
-	package "cooperation_element" - includes packages "apartment", "contact", "cooperation", "house", "ownership", "user";
-	package “mail” – its current content plus content of package “email” plus content of package “template”.